src/stx/common/trex_rx_timesync.h

```diff
@@ -42,12 +44,14 @@ class RXTimesync {
 
     Json::Value to_json() const;
 
+    bool hardware_timestamping_enabled;
+
   private:
     TimesyncPacketParser_err_t parse_ptp_pkt(uint8_t *pkt, uint16_t len, uint16_t rx_tstamp_idx, int port);
     void hexdump(const unsigned char *msg, uint16_t len); // TODO remove
-    void parse_sync(uint16_t rx_tstamp_idx, timespec *t);
+    void parse_sync(int port, uint16_t rx_tstamp_idx, timespec *t);
```

Comment:
I'd rather prefer this `int port` argument at the end of arguments' list.  This would make all `parse_*` more alike.  (but that could be a NIT)

Reply:
done